dell/inspiron/5509/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, ... }:
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/ssd
+  ];
+
+  # Essential Firmware
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  # Cooling Management
+  services.thermald.enable = lib.mkDefault true;
+
+  services.xserver = {
+    # GPU Driver Setting
+    videoDrivers = lib.mkDefault [ "intel" ];
```

Comment:
We usually modsettings for intel these days. Any reason the intel xorg driver is required here?

Reply:
Sorry, I had just copy-pasted it without realizing it, even my original configuration uses `modesetting`, I have changed it now!